components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityCoreConstants.java

```diff
@@ -36,6 +36,8 @@
     public final static String MULTI_ATTRIBUTE_SEPARATOR = "MultiAttributeSeparator";
     public final static String MULTI_ATTRIBUTE_SEPARATOR_DEFAULT = ",,,";
 
+    public static final String XML_SIGNATURE_ALGORITHM_URI_RSA_SHA1 = "http://www.w3.org/2000/09/xmldsig#rsa-sha1";
```

Comment:
A new dependency has to be introduced. If it is needed for other things we can consider using IdentityApplicationConstants. At the moment it is good to use a constant in the core.